sdk-core/src/main/java/care/data4life/sdk/ApiService.java

```diff
@@ -245,12 +293,37 @@ Completable deleteRecord(String alias, String recordId, String userId) {
                 .subscribeOn(Schedulers.io());
     }
 
+    /**
+     * Carry out needed logout actions.
+     * <p>
+     * When using refresh token, this will revoke the OAuth access.
+     * When using a static access token, nothing will be done.
```

Comment:
Why is that not possible?
From what I see, only the access token is used for ending the user's session in vega.

Reply:
It is the refresh token